src/scss/abstracts/_variables.scss

```diff
@@ -31,6 +31,8 @@ $navy: #203149;
 $navy-light: #2e405b;
 
 $blue-alt: #5c81a9;
+$blue-alt-contrast: #5879a0;
+$blue-on-yellow: #4d6b8d;
```

Comment:
Could you use another variable name, although this blue is so far only used with the yellow background, but itself is another tone of a blue that has nothing to do with the yellow.